### DISCUSSION TITLE
Specific composer options for dev/prod

### DISCUSSION THREAD

**Discussion by mrhackcz** [deployphp/deployer#3908] [Help needed] [answered]
Hi folks, do you know how to distinguish the `composer_options` variable for the production and development environment? I need to set `--no-dev` for production use without it in development. 
I tried using a selector on `set();` which doesn't work. 
Set label `'composer' => '--no-dev'` and use it in `set('composer_options')` with `get('labels')['composer']`, which also doesn't work.

Any recommendation ? Thank you!

**Comment by antonmedv** [comment, marked as answer]
```php
host('prod')
    ->set('composer_options', '--verbose --prefer-dist --no-progress --no-interaction --no-dev --optimize-autoloader');

host('dev')
    ->set('composer_options', '--verbose --prefer-dist --no-progress --no-interaction --optimize-autoloader');
```

**Reply by mrhackcz** [reply]
Thank you. Didnt know, that I can set host-specific variables.
